src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1663,6 +1663,11 @@ type RideHailingEstimate {
     productName: String
 }
 
+"""
+An ISO-8601-formatted local date, i.e. `2024-05-24` for the 24th of May, 2024.
+"""
+scalar LocalDate
```

Comment:
We could use the `@specifiedBy` annotation here but did we decide to just support a subset of the ISO-8601-formatted local dates? If so, we should probably mention it.

Reply:
I added a @specifiedBy annotation that links to the offical spec in https://github.com/opentripplanner/OpenTripPlanner/pull/5869/commits/1734b8a61334f4bcf3de140be163fd3f93910296

Problem is that you have to pay to get it. There are a handful of copies of the PDF floating around the internet but their legality is dubious, so I don't want to link it. I could also link to Wikipedia but that isn't a spec.

Reply:
I think it's ok to link it as it is the official source despite it being behind a pay wall.